jors_perprof-py.tex

```diff
@@ -56,9 +56,15 @@ \section{Availability}
 \section{Reuse potential}
 \input{reuse}
 
-\section*{\colorsectionnumberless{Acknowledgements}}
-We would like to thank the three anonymous reviewers who gave valuable insight
-and suggestions to this paper.
+
+\section*{\colorsectionnumberless{Acknowledgments}}
+The authors would like to thank FAPESP 
+(grants 2008/ 09685-8 and 2009/17273-4) and CNPq (grant 501763/ 2013-9) for the
```

Comment:
I know, but if it's kept, the final result is outside textwidth. Should we insert a footnote?